coinbaseadvanced/models/products.py

```diff
@@ -20,6 +20,7 @@ class ProductType(Enum):
 
     SPOT = "SPOT"
 
+Gran = {"ONE_MINUTE": 1, "FIVE_MINUTE": 5, "FIFTEEN_MINUTE": 15, "THIRTY_MINUTE": 30, "ONE_HOUR": 60, "TWO_HOUR": 120, "SIX_HOUR": 360, "ONE_DAY": 720}
```

Comment:
Should `ONE_DAY` be `1440` instead(`24*60=1440`)?

Reply:
Yup.